horace_core/algorithms/gen_sqw.m

```diff
@@ -753,7 +753,7 @@ function report_nothing_to_do(spe_only,spe_exist)
         data_ranges = data_ranges(f_valid_exist);
         data_range = data_ranges{1};
         for i=2:numel(pix_ranges)
-            data_range = mimnax_ranges(data_range,data_ranges{i});
+            data_range = minmax_ranges(data_range,data_ranges{i});
```

Comment:
Should this be in this PR?

Reply:
yes because it was a  bug and I've hit it.

Reply:
and I am refactoring `gen_sqw` (in next steps of this ticket)

Reply:
Please submit this as a separate quick-fix so we can pull it.

The reason for this is: say hypothetically we wanted to reject this PR or revert it at some point in the future. 

What happens to this fix? 
It:
- Is not documented in an issue
- Is hidden in a 56-file PR
- Where the PR's main thrust is not to fix this
- And this is not mentioned in the PR's description as fixing this
So we will have no record of it

Splitting different fixes out into their own PR reduces the load on future work, gets essential fixes like this into `master` faster meaning everyone can benefit, and makes the reviewers' job easier meaning that this PR can get through review faster.